localstack/services/firehose/firehose_api.py

```diff
@@ -122,7 +134,9 @@ def process_records(records, shard_id, fh_d_stream):
 
 
 def create_stream(stream_name, delivery_stream_type='DirectPut', delivery_stream_type_configuration=None,
-                  s3_destination=None, elasticsearch_destination=None):
+                  s3_destination=None, elasticsearch_destination=None, tags=None):
+    if tags is None:
```

Comment:
nit: could be simplified to `tags = tags or {}`

Reply:
More concise, thanks for the tip! Done.